packages/office-addin-dev-settings/src/registry.ts

```diff
@@ -184,7 +184,7 @@ export async function getStringValue(key: RegistryKey, value: string): Promise<s
 }
 
 export async function getValue(key: RegistryKey, value: string): Promise<RegistryValue | undefined> {
-  return new Promise<RegistryValue>((resolve, reject) => {
+  return new Promise<RegistryValue | undefined>((resolve, reject) => {
```

Comment:
Why is `| undefined` being added here?

Reply:
Because 3 lines below there's `resolve(undefined);`

Reply:
OK, this is fine. Sorry, I was reviewing too quickly and didn't see the outer context was returning undefined.